### PR TITLE
Deprecate Animation and Slider components

### PR DESCRIPTION
**Pull request by Klynger**
#### What problem is this solving?

We don't use these components anywhere and we don't recommend using it. Also, there is no blocks associate with any of them

### CODE REVIEW

File: docs/Slider.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Slider
+# Slider ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)
+
+:warning: **The Product Price block has been deprecated in favor of the [Slider Layout](https://vtex.io/docs/app/vtex.slider-layout) app**. Although support for this component is still granted, we strongly recommend you to update your store theme with the Slider Layout's blocks in order to keep up with the component's evolution.
```
Comment (guerreirobeatriz) [inline docs/Slider.md:3]:
```suggestion
:warning: **The Slider block has been deprecated in favor of the [Slider Layout](https://vtex.io/docs/app/vtex.slider-layout) app**. Although support for this component is still granted, we strongly recommend you to update your store theme with the Slider Layout's blocks in order to keep up with the component's evolution.
```

File: docs/Animation.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
-# Animation
+# Animation ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)
 
 ## Description
 
+:warning: **The Product Price block has been deprecated**. Although support for this block is still granted.
+
```
Comment (guerreirobeatriz) [inline docs/Animation.md:6]:
```suggestion
:warning: **The Animation block has been deprecated** (support is still granted).

```

File: docs/Animation.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
-# Animation
+# Animation ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)
 
 ## Description
 
+:warning: **The Product Price block has been deprecated**. Although support for this block is still granted.
```
Comment (dvm1984) [inline docs/Animation.md:5]: **The Product Price block has been deprecated**, although support for this block is still given.

File: docs/Animation.md (hunk 2 above)
Comment (dvm1984) [reply docs/Animation.md:6]: :warning: **The Animation block has been deprecated** (support is still given).

File: docs/Slider.md (hunk 1 above)
Comment (dvm1984) [reply docs/Slider.md:3]: Although support for this component is still given, it's strongly recommended that you update your store theme with the Slider Layout's blocks in order to keep up with the component's evolution.

**Pull request merged**